src/server/game/Mails/Mail.cpp

```diff
@@ -100,6 +100,10 @@ void MailDraft::prepareItems(Player* receiver, SQLTransaction& trans)
 
     m_mailTemplateItemsNeed = false;
 
+    // The mail sent after turning in the quest The Wrath of Neptulon contains 100g
+    if (m_mailTemplateId == 123)
```

Comment:
maybe there is a more generic way to specify money for specific mail template id ?

Reply:
Is it really directly attaching gold? Blizz tends to create bags with gold loot for this purpose

Reply:
The closest info I found is from [Wowwiki](https://wowwiki.fandom.com/wiki/Quest:The_Good_News_and_The_Bad_News) and [Wowhead](https://www.wowhead.com/quest=8729) where nobody mentions a bag. Also tried to search in item_template (the entries close to the quest items, as Blizzard usually does them sequentially) for potential items, but nothing.

I thought about making this a table (mail_money_reward) that takes mailTemplateId and the amount of coppers to attach. Is it worth it to have a whole table for it when this is the only blizzlike use case for it?